e2e/e2e_suite_test.go

```diff
@@ -49,7 +49,7 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 			<-testshell.ExecuteInDir(projectDir, "make", "start-cluster").Done()
 		})
 	} else {
-		if _, found := os.LookupEnv("IKE_SKIP_CLUSTER_SHUTDOWN"); !found {
+		if _, found := os.LookupEnv("IKE_E2E_SKIP_CLUSTER_SHUTDOWN"); !found {
```

Comment:
The CLUSTER_DIR too?

Reply:
Was thinking about that one, but I am not entirely sure, as the script can be used not only by e2e tests.

Reply:
ok.